js/src/common/components/ModalManager.js

```diff
@@ -46,10 +50,23 @@ export default class ModalManager extends Component {
     this.$()
       .one('shown.bs.modal', readyCallback)
       .modal({
-        backdrop: dismissible || 'static',
-        keyboard: dismissible,
+        // 'static' means that a backdrop click doesn't dismiss the modal
+        backdrop: backdropDismissible ? true : 'static',
```

Comment:
Why is this change necessary?

Reply:
Oops, it isn't. I think this slipped through the net while I was trying other approaches.

I'll revert back to the `||` method, but keep the comment as it makes the reasoning behind `"static"` clearer.